src/app/modules/core/advertisement/components/card/card-top/card-main-container/card-main-container.component.scss

```diff
@@ -1,18 +1,7 @@
-mat-card {
-  width: 100%;
-  max-width: 950px;
+#card {
+  margin-bottom: 20px;
 }
 
-mat-card-content {
-  max-width: 100%;
-}
-
-mat-card-title {
-  margin: 10px;
-}
-
-.button {
-  margin: 2px;
-  width: 100px;
-  float: right;
+.mobile-spacing {
+  margin-top: 15px;
 }
```

Comment:
In modalità tablet bisognerebbe sistemare lo spazio tra l'immagine e le facilities

Reply:
Mi è saltata qualche proprietà css mi sa, sistemo!